.github/workflows/runtimes-matrix.json

```
@@ -16,15 +17,17 @@
     "is_relay": true,
     "blocktime": 6000,
     "benchmarks_exclude_extrinsics": "pallet_nomination_pools::apply_slash_fail",
-    "extra_args": "--disable-mbm-checks"
+    "extra_args": "--disable-mbm-checks",
+    "template": "benchmarking/template.hbs"
```

We have a similar setting for benchmarking `benchmarks_exclude_extrinsics`, so I would use the same syntax here:
```suggestion
    "benchmarks_xcm_template": "benchmarking/template.hbs"
```